src/pcgnslib.c

```diff
@@ -2165,3 +2165,71 @@ int cgp_array_multi_read_data(int fn, int *A, const cgsize_t *rmin,
 
   return CG_ERROR;
 }
+
+/*===== Boundary Condition Functions =============================*/
+
+int cgp_boco_write(int file_number, int B, int Z, const char * boconame,
```

Comment:
Why do you still need a cgp version? It would be better not to have a  cgp version for all the other APIs that write a point list.

Reply:
Ah, sorry. Yeah, that's left over. I wasn't intending to keep it. Fixed now.